lte/gateway/c/core/oai/test/amf/amf_app_test_util.cpp

```diff
@@ -659,8 +656,7 @@ bool check_ue_context_state(amf_ue_ngap_id_t ue_id,
   }
 
   m5gcm_state_t cm_state;
-  rc = amf_get_ue_context_cm_state(ue_id, &cm_state);
-  if (rc != RETURNok) {
+  if (amf_get_ue_context_cm_state(ue_id, &cm_state) != RETURNok) {
```

Comment:
Add an error log. Helps in debugging

Reply:
"lte/gateway/c/core/oai/test/amf/amf_app_test_util.cpp" this file is part of unit test framework and logging functionalities are not available while running the 5G unit test framework.